iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AliasesXml.java

```diff
@@ -45,6 +47,16 @@ public static void read(Identifiable<?> identifiable, NetworkXmlReaderContext co
         identifiable.addAlias(alias, aliasType[0]);
     }
 
+    public static <T extends Identifiable> void read(List<Consumer<T>> toApply, NetworkXmlReaderContext context) throws XMLStreamException {
+        if (!context.getReader().getLocalName().equals(ALIAS)) {
+            throw new IllegalStateException();
+        }
+        String[] aliasType = new String[1];
+        IidmXmlUtil.runFromMinimumVersion(IidmXmlVersion.V_1_4, context, () -> aliasType[0] = context.getReader().getAttributeValue(null, "type"));
+        String alias = context.getAnonymizer().deanonymizeString(context.getReader().getElementText());
+        toApply.add(identifiable -> identifiable.addAlias(alias, aliasType[0]));
```

Comment:
If we don't factorize `getSubElements`, we could factorize this with a method returning `identifiable -> identifiable.addAlias(alias, aliasType[0])`, which is then applied or not.